### PR TITLE
Lab2

### PR DESCRIPTION
**Pull request by marina01p**
2nd laboratory work finished

### CODE REVIEW

File: Laboratory1/app/src/main/java/com/example/laboratory1/ThirdActivity.kt [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+//                TODO("Not yet implemented")
+//            }
+//
+//            override fun onTextChanged(s: CharSequence?, start: Int, before: Int, count: Int) {
+//                val thisInput = yourTxtArea.text.toString().trim()
+//
+//                if (thisInput.isNotEmpty()) {
+//                goToListBtn.isEnabled
+//            }
+//        }})
```
Comment (ASV44) [inline Laboratory1/app/src/main/java/com/example/laboratory1/ThirdActivity.kt]: Why do you need that dead code? Remove it and you will find it in version control if you will need it later

**Review by ASV44 — CHANGES_REQUESTED** [review]
@marina01p Change init of all views, including init of activities views to android view binding https://developer.android.com/topic/libraries/view-binding

Remove all `findViewById` and change use of it to access of view elements to view binding access of elements

File: Laboratory1/app/src/main/java/com/example/laboratory1/ThirdActivity.kt (hunk 1 above)
Comment (marina01p) [reply Laboratory1/app/src/main/java/com/example/laboratory1/ThirdActivity.kt]: I refactored the code using viewBinding and removed all unnecessary, commented code. Thanks for the feedback!

**Review by ASV44 — APPROVED** [review]
@marina01p PR approved. Feel free to merge it.

**Pull request merged**